### PR TITLE
Removing Page 3 and links to it

### PR DESCRIPTION
**Pull request by JiggyPete**
This PR removes the third page of the stamp duty calculator.

**Page being removed**
screen shot 2016-11-28 at 12 11 16

Any navigation to Page 3 has also been removed.
**Page 2**
screen shot 2016-11-28 at 12 11 09 copy

### CODE REVIEW

**Comment by rnalexander** [comment]
This is *so* much easier to review!
:+1:

File: config/routes.rb [hunk 1]
```diff
@@ -20,27 +20,15 @@ def matches?(request)
   end
 
   resource :stamp_duty, path: 'stamp-duty-calculator' do
```
Comment (jongilbraith) [inline config/routes.rb]: Can these empty blocks go?

Comment (JiggyPete) [inline config/routes.rb]: Great point Jon! All updated now.

**Comment by JiggyPete** [comment]
@rnalexander thanks!

File: config/routes.rb (hunk 1 above)
Comment (jongilbraith) [inline config/routes.rb]: Sweet, cheers @JiggyPete

**Comment by jongilbraith** [comment]
👍

**Pull request merged**